scvi/dataset/dataset10X.py

```diff
@@ -186,7 +186,8 @@ def populate(self):
         else:
             gene_names = None
             for measurement_type in np.unique(measurements_info[2]):
-                measurement_mask = measurements_info[2] == measurement_type
+                # .values required to work with sparse matrices
+                measurement_mask = (measurements_info[2] == measurement_type).values
```

Comment:
I'v never seen `.values` for sparse matrix neither for numpy array; is it something like `todense()`?. I might miss something but are you sure it's working? and compatible with numpy array?

Reply:
`measurement_mask` is a pandas DF. For some reason (not sure why) if you use the slice of the data frame column to access columns of the sparse matrix, the shape returned was the same as it was before. `.values` is the pandas way of grabbing the underlying `np.ndarray` so this should be universal.

Reply:
not sure why we need this?

Reply:
```
In [144]: test = csr_matrix(np.zeros((100, 3)))                                                                                                                                                                                    

In [145]: mask = pd.DataFrame(data=np.array([True, False, True]))                                                                                                                                                                  

In [146]: test[:, mask[0]].shape                                                                                                                                                                                                   
Out[146]: (100, 3)

In [147]: test[:, mask[0].values].shape                                                                                                                                                                                            
Out[147]: (100, 2)
```

Reply:
uncool..